### PR TITLE
User edit

### PR DESCRIPTION
**Pull request by jasiek-net**

### CODE REVIEW

File: src/services/user/get.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+export default function get(req: Request, res: Response, next: NextFunction): any {
+  authenticate(req, res, next)
+    .then((user) => {
+      const {
+        firstname,
+        lastname,
+        email,
+        birthday,
+
```
Comment (michalmikolajczyk) [inline src/services/user/get.ts:19]: why the empty line?

File: src/tests/user/get.spec.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  phone: null,
+  emergency: null,
+  city: null,
+  country: null,
+  brief: null,
+
+  image: null,
+};
+
+describe('user/get tests', () => {
```
Comment (michalmikolajczyk) [inline src/tests/user/get.spec.ts:28]: please be precise in test naming

Comment (michalmikolajczyk) [reply src/tests/user/get.spec.ts:28]:
```
describe('/user GET', () => {
...
```

**Review by michalmikolajczyk — CHANGES_REQUESTED** [review]
dobra robota, drobne poprawki i wrzuta

**Comment by michalmikolajczyk** [comment]
please rebase on `master`

File: src/services/user/get.ts (hunk 1 above)
Comment (jasiek-net) [reply src/services/user/get.ts:19]: it's separate required information from optional (and image)

File: src/tests/user/get.spec.ts (hunk 2 above)
Comment (jasiek-net) [reply src/tests/user/get.spec.ts:28]: it's describe path to file, not request method (likewise in frontend test)

**Review by michalmikolajczyk — APPROVED** [review]

**Pull request merged**